requirements.txt

```diff
@@ -1,6 +1,4 @@
-requests==2.11.1
-gcloud==0.17.0
-oauth2client==3.0.0
-requests-toolbelt==0.7.0
-python-jwt==2.0.1
-pycrypto==2.6.1
\ No newline at end of file
+requests>=2.11.0
+gcloud>=0.17.0
```

Comment:
I would strongly recommend not depending on this package *at all*. It's not been updated in quite some time. I'd switch to depend on the individual `google-cloud-{thing}` packages.

Reply:
Good point. Done.